[0m16:16:42.303945 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f42fca74b00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f42fc856d80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f42fe03dca0>]}


============================== 16:16:42.307742 | accc898d-17fc-45f4-886d-a4a6e1129424 ==============================
[0m16:16:42.307742 [info ] [MainThread]: Running with dbt=1.8.7
[0m16:16:42.308282 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/me/.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt parse', 'send_anonymous_usage_stats': 'True'}
[0m16:16:42.308689 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /home/me/dagster_dbt_tutorial/dagster-and-dbt/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m16:16:42.309744 [debug] [MainThread]: Resource report: {"command_name": "parse", "command_wall_clock_time": 0.061163493, "process_user_time": 1.229706, "process_kernel_time": 0.277675, "process_mem_max_rss": "97312", "process_in_blocks": "21808", "process_out_blocks": "8", "command_success": false}
[0m16:16:42.310330 [debug] [MainThread]: Command `dbt parse` failed at 16:16:42.310220 after 0.06 seconds
[0m16:16:42.310713 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f42fc3923f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f42fc3925d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f42fc654fe0>]}
[0m16:16:42.311078 [debug] [MainThread]: Flushing usage events
[0m16:16:47.836782 [debug] [MainThread]: Error sending anonymous usage statistics. Disabling tracking for this execution. If you wish to permanently disable tracking, see: https://docs.getdbt.com/reference/global-configs#send-anonymous-usage-stats.
